clearwater-sip-stress-coreonly.root/usr/share/clearwater/sip-stress/callee_redirection.xml

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
+<!DOCTYPE scenario SYSTEM "sipp.dtd">
+
+<scenario name="Clearwater callee">
+  <recv request="INVITE" rrs="true" crlf="true">
+    <action>
+      <!-- Save off our identity for later use -->
+      <ereg regexp=".*<sip:(.*)@(.*)>.*" search_in="hdr" header="To:" check_it="true" assign_to="_,dn,home_domain" />
+    </action>
+  </recv>
+
+  <nop>
+    <action>
+      <!-- Save off the per-transaction headers, for use on the eventual 180 Ringing and 200 OK -->
+      <assignstr assign_to="invite_cseq" value="[last_Cseq:]" />
+      <assignstr assign_to="invite_via" value="[last_Via:]" />
+    </action>
+  </nop>
+
```

Comment:
Can you add a comment here (comments are of the form `<!-- Save off our identity for later use -->`) explaining why we have two 302s? (i.e. to emulate the nested redirection case and increase BGCF load)